src/unix/netbsd.c

```diff
@@ -92,6 +93,22 @@ int uv_exepath(char* buffer, size_t* size) {
 
   /* Set new size. */
   *size = strlen(buffer);
+#else
+  ssize_t n;
```

Comment:
don't we have this code somewhere else too? maybe we can DRY it?

Reply:
It's in `src/unix/procfs-exepath.c`.

Reply:
Suggestions what I should do here?

Reply:
Use the function defined there as the fallback, instead of duplicating it here, if possible.

Reply:
You know your build system best - what in particular do you recommend?

Reply:
I think you could re-arrange that file into being header, so we can `#include` it on the necessary platforms.